client/adventureMap/CList.cpp

```diff
@@ -262,6 +267,43 @@ std::string CHeroList::CHeroItem::getHoverText()
 	return boost::str(boost::format(CGI->generaltexth->allTexts[15]) % hero->getNameTranslated() % hero->type->heroClass->getNameTranslated());
 }
 
+void CHeroList::CHeroItem::gesture(bool on, const Point & initialPosition, const Point & finalPosition)
```

Comment:
Looks like copy-paste from towns list. Maybe introducing base class common for both with accepting swap function as an argument?

Reply:
Yes, may be possible with some work I think.
But I also think it would make sense to keep them separate. We have two slots left for the radial wheel. And we should keep the possibility to add different things for town and heroes.